src/20230926_AaveV3_Eth_GHOFunding/AaveV3_Ethereum_GHOFunding_20230926.sol

```diff
@@ -0,0 +1,116 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+import {AaveV2Ethereum, AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+import {AaveSwapper} from 'aave-helpers/swaps/AaveSwapper.sol';
+import {AaveMisc} from 'aave-address-book/AaveMisc.sol';
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {SafeERC20} from 'solidity-utils/contracts/oz-common/SafeERC20.sol';
+
+interface aTokenV1 {
+  function redeem(uint256 _amount) external;
+}
+
+/**
+ * @title GHO Funding
+ * @author TokenLogic
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0xb094cdc806d407d0cf4ea00e595ae95b8c145f77b77cce165c463326cc757639
+ * - Discussion: https://governance.aave.com/t/arfc-treasury-management-gho-funding/14887
+ */
+contract AaveV3_Ethereum_GHOFunding_20230926 is IProposalGenericExecutor {
+
+  struct Asset {
+    address underlying;
+    address aToken;
+    address oracle;
+    uint256 amount;
+  }
+
+  address public constant COLLECTOR = address(AaveV3Ethereum.COLLECTOR);
+  address public constant MILKMAN = 0x11C76AD590ABDFFCD980afEC9ad951B160F02797;
+  address public constant PRICE_CHECKER = 0xe80a1C615F75AFF7Ed8F08c9F21f9d00982D666c;
+  address public constant GHO_ORACLE = 0x3f12643D3f6f874d39C2a4c9f2Cd6f2DbAC877FC; 
+
+  function execute() external {
```

Comment:
why is this so unnecessarily imprecise?

The forum post & snapshot said 1.6M
the updated(after aip) post now says 1.611M
if it was executed **now**, it was already 1.613M

Wouldn't it be better to just be exact? as there's not really a good reason not to be - you already have all the amount and could just make e.g. DAI on v3 fill the 1.6M.